ci/v2/lint.sh

```diff
@@ -28,6 +28,13 @@ annotations() {
   "${ROOT_DIR}"/lint/check_api_annotations.py
 }
 
+run_semgrep() {
+  # semgrep 1.32.0 requires rich>=12.6.0, but we install rich 12.0.1 which is 
```

Comment:
curious on where are we installing rich 12.0.1? can we upgrade that place to 12.6.0 instead?

otherwise does it work if you put this into lint-requirements.txt

Reply:
Looks like it's coming from `rllib-requirements.txt`. It pins `rich` to `12.0.1`. I've upgraded the version there.